cogs/profile.py

```diff
@@ -240,13 +240,15 @@ async def view(self, ctx: commands.Context):
     async def inventory(self, ctx: commands.Context):
         """Shows the items that you own."""
         user = await func.get_user(ctx.author.id)
-
         embed = discord.Embed(title=f"🎒 {ctx.author.display_name}'s Inventory", color=0x5cb045)
-        embed.description = f"```❄️ Starcandies        x{user['candies']}\n" \
-                            f"🌸 Rare rolls         x{user['roll']['rare']}\n" \
-                            f"💎 Epic rolls         x{user['roll']['epic']}\n" \
-                            f"👑 Legend rolls       x{user['roll']['legendary']}\n" \
-                            f"🎁 Gifts              x{user.get('gifts', 0)}\n\n"
+        embed.description = f"```{'❄️ Starcandies':<21} x{user['candies']}\n"
+
+        for tier, count in user.get("roll").items():
+            if tier in iufi.TIERS_BASE.keys():
+                emoji, _ = iufi.TIERS_BASE.get(tier)
+                embed.description += f"{emoji} {tier.title() + ' Rolls':<18} x{count}\n"
+
```

Comment:
Should we hide mystic and celestial rolls, since most of the times, it will be zero? can we make it in such a way that it only shows if count > 0 ?

Reply:
If I receive mystic roll once, then it will always show mystic roll in my inventory, right?

Reply:
oh yes i will add that also <3

Reply:
If I verify that the roll is greater than 0, another roll will not appear in the inventory if the count is 0.

Reply:
I meant for overall, 
if count == 0:
    continue;

so even rare and epic rolls won't be shown if user doesn't have those rolls

Reply:
oh i see. I prefer to keep rare and epic rolls in the inventory even when there aren't any, as it enhances the inventory view. However, I'm fine with removing everything from the inventory if necessary.